### ISSUE TITLE
Cheatsheets or guides for transitioning to R from another language

### ISSUE THREAD

**Issue by gaborcsardi** [RConsortium/wishlist#5]
These could be printable one (or more?) page documents, and/or websites.

Candidate languages:
- Python
- Matlab
- C (?)
- Java (?)
- SAS (?)
- SPSS (?)
- Perl (?)
- SQL (?)

Whether it is a website or downloadable PDFs, the sources should be open source at GitHub, and the docs can be at R-hub.

**Comment by leeper** [comment]
Also Stata.

**Comment by karthik** [comment]
It would be nice to prioritize by languages from which people are likely to transition. Stata, SAS, and SQL users are more likely on the analyst end of the spectrum.

**Comment by gaborcsardi** [comment]
@karthik True. But I think there are also a lot (?) of people that know some Python or Matlab, etc., i.e. not the typical stats languages, and they are moving into data science. Now their choice is to learn (more) DS and stats in Python or Matlab, or give R a try.

Anyway, prioritizing is indeed important, and maybe it is possible to collect some data that can drive it. Don't know where or how, though......

**Comment by AmeliaMN** [comment]
I'm working (slowly...) on a cheatsheet that is a comparison of what I see as the three main R syntaxes: 

- dollar sign/base R: `plot(mtcars$mpg, mtcars$cyl)`
- formula/lattice/mosaic: `xyplot(cyl~mpg, data=mtcars)`
- tidyverse: `ggplot(mtcars) + geom_point(aes(x=mpg, y=cyl))`
